### PR TITLE
Address

### PR DESCRIPTION
**Pull request by chanchal16**
- Created address management
- add address
- edit address
- remove address
- change address

Live Preview: https://relaxed-zuccutto-bcf6e7.netlify.app/

### CODE REVIEW

File: src/backend/controllers/AddressController.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+/**
+ * All the routes related to Cart are present here.
+ * These are private routes.
+ * Client needs to add "authorization" header with JWT token in it to access it.
+ * */
+
+/**
+ * This handler handles getting items to user's address.
+ * send GET Request at /api/user/address
+ * */
```
Comment (pradipgharge) [inline src/backend/controllers/AddressController.js:13]: remove commented code

File: src/styles/address.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    gap: 1rem;
+    margin: auto;
+    padding: 1rem;
+}
+/* address card */
+.address-card{
+    width: 300px;
```
Comment (pradipgharge) [inline src/styles/address.css:78]: use rems for length units

**Review by pradipgharge — APPROVED** [review]
code looks clean and good to merge, just remove the commented code

**Review by sandeepkumardev — APPROVED** [review]

File: src/backend/controllers/AddressController.js (hunk 1 above)
Comment (chanchal16) [reply src/backend/controllers/AddressController.js:13]: It's docs for a better understanding of the code.

**Pull request merged**